server/api/observation/observation.model.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, Unique } from 'typeorm';
+import { ObservationType } from './type/observation-type.model';
+
+/**
+ * Observation are created when users self-report COVID-19 symptoms via https://mystatus.mil/
+ * This data is then replicated from the original storage to this entity.
+ */
+@Entity()
+@Unique(['id', 'documentId'])
+export class Observation extends BaseEntity {
+
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  // ID of the original document
+  @Column({ length: 100 })
+  documentId: string;
+
+  /* DOD unique ID that is associated with personnel and their CAC card.
+  We don't maintain the list of these IDs so there is no integrity constraint (foreign key).
+  */
+  @Column({ length: 10 })
+  edipi: string;
+
+  /* timestamp when the observation was reported
+  TODO: https://app.asana.com/0/1198942245525292/1200506195976678/f
+   */
+  @Column({ length: 50 })
+  timestamp: string;
```

Comment:
This timestamp would be very similar to the Orphaned Record timestamp (it's created from the same source DynamoDB data. Copying the declaration here for convenience.
```js
  @Column({
    type: 'timestamp',
    transformer: timestampColumnTransformer,
  })
  timestamp!: Date;
```

Reply:
Thanks! Done.